tedana/resources/decision_trees/kundu.json

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
YAML files are easier to format and allow comments. They might be preferable to JSONs for the trees.

Reply:
This would involve a few substantial changes... I'm not convinced that YAML is worth the switch. You can always add a comment by making a field called `_comment`, which would get ignored in the current code.